content/_includes/post.macros.njk

```diff
@@ -349,11 +349,9 @@ params:
 
     {{ client(post.data.client) }}
 
-    {% if dateformat %}
-      {% set events = post | pageEvents if ('_talk' in (post.data.tags or [])) else none %}
-      {% if not (events and events.length > 0) %}
```

Comment:
@mirisuzanne Is there a reason why you were checking for events here instead of just checking if it's a talk/workshop? I didn't see any practical difference.

Reply:
no good reason, this change looks good to me.